vite.config.ts

```diff
@@ -0,0 +1,34 @@
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => ({
+  define: {
+    APP_VERSION: JSON.stringify(process.env.npm_package_version),
+    ENV: JSON.stringify(process.env.ENV),
+    BASE_SIGNING_URL: JSON.stringify(process.env.BASE_SIGNING_URL),
+    BTC_COIN: JSON.stringify(process.env.BTC_COIN),
+    BTC_DEFAULT_FEE_RATE: JSON.stringify(process.env.BTC_DEFAULT_FEE_RATE),
+    BTC_TX_BASE_URL: JSON.stringify(process.env.BTC_TX_BASE_URL),
+    BTC_TESTNET: JSON.stringify(process.env.BTC_TESTNET),
+    KEYRING_LOGOUT_MS: JSON.stringify(process.env.KEYRING_LOGOUT_MS),
+    LATTICE_CERT_SIGNER: JSON.stringify(process.env.LATTICE_CERT_SIGNER),
```

Comment:
I suspect you have it covered, but just want to point out that some of these are named differently than before (`REACT_APP_*` prefix has been removed) just in case you haven't accounted for that in the external config file